### ISSUE TITLE
Simpler version of CSV export for accounting data only

### ISSUE THREAD

**Issue by alanna** [opencollective/opencollective#5670]
## User story 
User feedback: "If the monthly income and expenditure spreadsheets were presented in a more formal accounting style, without all the additional extraneous columns."

## Best solution for this problem
Maybe a way for users to select or unselect the columns to be included on their CSV download? Or choices like "everything" and "accounting data only". Accounting data columns would be amount (with '-' in front for debits), date, balance.

#4396 is the larger context, about how OC exports could better align with accounting formats. #3875 is currently being worked on which will help a lot.

## MVP
Users can of course just delete unwanted columns after downloading, so I guess that's the workaround.

**Comment by znarf** [comment]
<img width="302" alt="Screenshot 2022-06-16 at 09 55 17" src="https://user-images.githubusercontent.com/806/174021211-19f58787-265b-49b9-bffe-dce8c52f4d5f.png">

Today we have v1 and v2, v1 is deprecated and will disappear soon. This can be an opportunity to feature 2 buttons like: "simple CSV (new)" and "detailed CSV".

**Comment by poohlaga** [comment]
Would be nice in the detailed view to include the ID of the transaction so you can build the URL like '85809' to https://opencollective.com/opensource/expenses/85809

**Comment by poohlaga** [comment]
Also, I keep hitting a 1000 row limit on CSV export. Is there a way around that to get more data out of there?

**Comment by alanna** [comment]
Yeah for this to work the exports must be comprehensive not cut off.

**Comment by znarf** [comment]
> Also, I keep hitting a 1000 row limit on CSV export. Is there a way around that to get more data out of there?

From the web interface, pick a timeframe that is less than 2 months and you will have everything in that timeframe.

> Would be nice in the detailed view to include the ID of the transaction so you can build the URL like '85809'

For this, you need to construct the CSV URL manually and add the `expenseLegacyId` field:

E.g.: https://rest.opencollective.com/v2/opensource/transactions.csv?add=expenseLegacyId

We have some advanced documentation here:
https://github.com/opencollective/opencollective-rest/blob/main/docs/transactions.md

**Comment by Miragide** [comment]
Bonjour, en tant que collectif admin, en allant sur /transactiosn > Download export, je m'attends à obtenir un document qui me permet de recalculer la balance par défaut.
Or, par défaut, toutes les lignes de données ne s'affichent pas dans le document exporté, ce qui ne permet pas de comprendre tous les chiffres. Le moyen d'activer toutes les "catégories" de ligne de données n'est pas clair.
Je pense que toutes les lignes de données devraient être exportées par défaut, pour éviter la confusion.

Avant, l'export CSV était disponible depuis la partie ADMIN, ce qui était plus simple à trouver.
La page transaction est plus difficile à trouver que la page admin.
Ce serait utile de pouvoir exporter les données depuis la page d'administration également, comme avant.

Aussi, un comptable ne comprend pas pourquoi les entrées et sorties sont dans la même colonne. Il me semble que ces données sont dans des colonnes séparées - habituellement - dans les logiciels comptables.

Merci par ailleurs pour les améliorations apportées à la fonctionnalité d'export et pour votre travail.
